README.md

```diff
@@ -15,7 +15,13 @@ A document and demo of the DRA support for GPUs provided by this repo can be fou
 This section describes using `kind` to demo the functionality of the NVIDIA GPU DRA Driver.
 
 First since we'll launch kind with GPU support, ensure that the following prerequisites are met:
-1. `kind` is installed. See the official documentation [here](https://kind.sigs.k8s.io/docs/user/quick-start/#installation).
+1. `kind` is installed. See the official documentation [here](https://kind.sigs.k8s.io/docs/user/quick-start/#installation). Note that there is a [known issue with kind](https://kind.sigs.k8s.io/docs/user/known-issues/#pod-errors-due-to-too-many-open-files). You may see the error while trying to tail the log of a running pod in a kind cluster: `failed to create fsnotify watcher: too many open files.` The issue can be resolved by increasing the  default values for the configuraton parameters in `/proc/sys/fs/inotify`.
+```console
+ sudo sysctl -w fs.inotify.max_user_watches=<a larger value>
+ sudo sysctl -w fs.inotify.max_user_instances=<a large value>
+ sudo sysctl -w fs.inotify.max_queued_events=<a large value>
+```
+
```

Comment:
Let's leave out the solution, and just include the link (the solution is well documented in the link itself).

Can we also move this note to after all of the instructions for installing the prerequisites. I feel like it distracts from the flow to have thid detail here, and it can be done after `kind` is installed (it doesn't have to be done together with it).

Reply:
Moved it after the `kubectl log` command, which will provide the right context.

Reply:
I've applied the suggested change. I put it here because line 133 (right before this) is a command showing the pod log, which provides the right context. I'm okay with moving it to the beginning of this section too.

Reply:
Does it work to update `fs.inotify.max_user_watches`*after* the kind cluster has been started? If so, then let's move this to line 106, i.e. *before* the log calls are made (so people see the warning before things suddenly aren't working).

If you have to update  `fs.inotify.max_user_watches` *before* the kind cluster is started, then I still think my original suggestion is the right place to put this (i.e. before the kind cluster is started).

Reply:
> "Does it work to update fs.inotify.max_user_watchesafter the kind cluster has been started?"
Yes, that's what I did. I didn't need to recreate the kind cluster.  How about moving it line 96?